app/scripts/controllers/user-storage/auth-snap-requests.ts

```diff
@@ -0,0 +1,21 @@
+import type { SnapId } from '@metamask/snaps-sdk';
+import type { HandleSnapRequest } from '@metamask/snaps-controllers';
+import { HandlerType } from '@metamask/snaps-utils';
+
+type SnapRPCRequest = Parameters<HandleSnapRequest['handler']>[0];
+
+const snapId = 'npm:@metamask/message-signing-snap' as SnapId;
+
+export function createSnapSignMessageRequest(
```

Comment:
This looks like a duplicate of code in the other PR. Will these be deduplicated once one PR is merged?

Reply:
Yep exactly! 😄

Reply:
Done, this is reusing the existing functions from the auth controller.